js/webtorrent/components/torrentViewer.js

```diff
@@ -36,6 +48,24 @@ class TorrentViewer extends React.Component {
       mainButtonId = 'startDownload'
     }
 
+    if (torrentIdProtocol === 'magnet:') {
+      saveButton = (
+        <Button
+          l10nId='copyMagnetLink'
+          className='whiteButton copyMagnetLink'
+          onClick={() => dispatch('copyMagnetLink')}
+        />
```

Comment:
Magnet links get a "Copy Magnet Link" button, instead of "Save Torrent File" which made no sense.